python/graphscope/gsctl/commands/common.py

```diff
@@ -82,5 +82,70 @@ def close():
             info("Try 'gsctl --help' for help.")
 
 
+disclaimer = """
+Disclaimer: The `estimate` command is a roughly estimation of the memory usage of the graphscope engine.
+The actual memory usage may vary vastly due to the complexity of the graph algorithm and the data distribution.
+
+The estimation is based on the following formulas:
+let #V = number of vertices, #E = number of edges,
+assume load_factor of hashmap is 0.41, sizeof(inner_id) = sizeof(edge_id) = sizeof(vertex_id) = 8 bytes,
+where vertex_id is the original representation of vertex, it could be string though, in this case, the memory usage will grow.
+Graph:
```

Comment:
- Move this as comments of the function for internal reference. No need to explain in such detail. 
- Spell out the assumption when you estimate, not how to estimate....
     - Assuming the graph is a simple graph, without labels, properties, and features...
     - Assuming the graph algorithms you are going to run will not generate a huge amount of intermediate data, e.g., PageRank, SSSP, CC, not algorithms like Louvain...